### ISSUE TITLE
Update chpldoc `usage` section to include `import`

### ISSUE THREAD

**Issue by bradcray** [chapel-lang/chapel#15327]
Now that we have an `import` statement, we should probably update the `chpldoc` usage section generated for each module to have two lines, for example:

```chapel
import Module.[...];`
```
or:
```chapel
use Module ...;
```

**Comment by bradcray** [comment]
I labeled this as "easy", but I think there's at least a minor design hurdle to decide how to format the messages above...

**Comment by souris-dev** [comment]
I would like to work on this issue. 

For the design hurdle, I think that one way to go about that is first adding information about the new `import` statement in the Modules primer, and then adding this to the usage section of each module (before or after the standard `use` statement syntax):
``` chapel
import <Module name>.[variable/procedure...];
```
"For more information on the `import` statement, please see: <link>." could also be added with it in case the person reading it is not familiar with `import`.

**Comment by bradcray** [comment]
Hi @souris-dev: I agree that we need to add documentation on `import`, and that is currently already on  our TODO list.  I would suggest not having every chpldoc'd module point to that documentation because I think we need to optimize it for the typical user rather than the beginning user.  Maybe put another way:  If we had every reference to any Chapel feature point to how to learn more about that feature, the content would be bogged down by those pointers.  Better would be to document everything clearly and then rely on search to help people figure out "what is this `import` or `use` thing anyway?"

That said, I like your proposal (though we might want to say `.[symbol]` or `.[identifier]` since it could be arbitrary things other than variables/procedures being imported (e.g., types, enums, sub-modules, etc.).  And I think we can fine-tune the wording/formatting pretty easily once the implementation is there.

I'm not aware of anyone else working on this if you'd like to run with it..

**Comment by souris-dev** [comment]
I understand what you're trying so say with letting people use the search function to look up what it is supposed to do and I'll leave that part out in the usage description.
And as you said, there could be other arbitrary things being imported, I'll go with:
```chapel
import <Module name>.[symbol];
```
I'm working on it, and should be ready to create a PR soon.

**Comment by lydia-duncan** [comment]
Looks like this issue can be closed now?

**Comment by bradcray** [comment]
Yep, thanks, resolved by @souris-dev in #15411
